packages/tatum-bsc/src/wallet/wallet.ts

```diff
@@ -31,11 +31,11 @@ export const generateBscWallet = async (testnet: boolean, mnem: string): Promise
 
 /**
  * Generate wallet
- * @param testnet testnet or mainnet version of address
  * @param mnemonic mnemonic seed to use. If not present, new one will be generated
+ * @param options.testnet optional testnet or mainnet version of address
  * @returns wallet or a combination of address and private key
  */
-export const generateWallet = (testnet: boolean, mnemonic?: string) => {
-  const mnem = mnemonic ? mnemonic : generateMnemonic(256)
-  return generateBscWallet(testnet, mnem)
+export const generateWallet = (mnemonic: string, options?: { testnet?: boolean }) => {
```

Comment:
mnemonic should be optional

Reply:
Why? In master I can see required.
https://github.com/tatumio/tatum-js/blob/d1de9b125a1b03ed72a6b3676dcadde9bb64891b/src/wallet/wallet.ts#L163